doc/classes/String.xml

```diff
@@ -522,6 +535,21 @@
 				Returns [code]true[/code] if this string represents a well-formatted IPv4 or IPv6 address. This method considers [url=https://en.wikipedia.org/wiki/Reserved_IP_addresses]reserved IP addresses[/url] such as [code]"0.0.0.0"[/code] and [code]"ffff:ffff:ffff:ffff:ffff:ffff:ffff:ffff"[/code] as valid.
 			</description>
 		</method>
+		<method name="is_valid_unicode_identifier" qualifiers="const">
+			<return type="bool" />
+			<description>
+				Returns [code]true[/code] if this string is a valid Unicode identifier.
+				[codeblock]
+				print("node_2d".is_valid_unicode_identifier())    # Prints true
+				print("TYPE_FLOAT".is_valid_unicode_identifier()) # Prints true
+				print("1st_method".is_valid_unicode_identifier()) # Prints false
+				print("MyMethod#2".is_valid_unicode_identifier()) # Prints false
+				print("体力".is_valid_unicode_identifier())       # Prints true
+				[/codeblock]
+				See also [method is_valid_ascii_identifier].
+				[b]Note:[/b] This method performs basic Unicode identifier checking, which is the same as how GDScript parses identifiers. See [method TextServer.is_valid_identifier] for detailed rules and potential advanced checks that utilize other Unicode features.
```

Comment:
Nitpick. I feel like this note can be shortened. I also get you're trying to direct readers interested in the "advanced checks", but it's a lot of alienating 
```suggestion
				[b]Note:[/b] This method performs basic Unicode identifier checking, which is the same as how GDScript parses identifiers. See [method TextServer.is_valid_identifier] for more details.
```
Or:
```suggestion
				[b]Note:[/b] This method checks identifiers the same way as GDScript. See [method TextServer.is_valid_identifier] for more advanced checks.
```
Or somewhat similar.